src/tk/wurst_client/mods/BaseFinderMod.java

```diff
@@ -60,94 +69,51 @@ public void onUpdate()
 		if(hasTimePassedM(3000))
 		{
 			matchingBlocks.clear();
+			
+			int playerX = (int) Math.floor(mc.thePlayer.posX);
+			int playerY = (int) Math.floor(mc.thePlayer.posY);
+			int playerZ = (int) Math.floor(mc.thePlayer.posZ);
+
 			for(int y = range; y >= -range; y--)
-			{
 				for(int x = range; x >= -range; x--)
-				{
 					for(int z = range; z >= -range; z--)
 					{
-						int posX = (int)(mc.thePlayer.posX + x);
-						int posY = (int)(mc.thePlayer.posY + y);
-						int posZ = (int)(mc.thePlayer.posZ + z);
+						int posX = playerX + x;
```

Comment:
"There is a difference between a variable that is created and destroyed 1024 times"
That's absolutly true, but only happen for fields (members of classes).

Local variables (variables inside of functions) aren't created and destroyed.
tl;dr Both versions will Compile to the same bytecode.

local variables are represented in JVM bytecode like an array.
This array is created as soon as the method is entered and will get destroyed if the method is exited.
The compiler will assign an index for each local variable.
jvm bytecode does not have local variable declarations.

So if we take the following code:
{
    for(int i = 0; i < 10; i ++) {
        int x = 20;
        int y = 30;
        int z = 40;
        System.out.println(x + y + z);
    }
}

You can say (simplified) that it will compile to something like that:
{
    Object[] locals = new Object[4];
    for(locals[0] = 0; locals[0] < 10; locals[0]++) {
        locals[1] = 20;
        locals[2] = 30;
        locals[3] = 40;
        System.out.println(locals[0] + locals[1] + locals[2];
    }
}

Now you should be able to understand that there's not difference.